tracer/test/Datadog.Trace.Security.IntegrationTests/Rcm/AspNetMvc5AsmRulesToggle.cs

```diff
@@ -0,0 +1,111 @@
+// <copyright file="AspNetMvc5AsmRulesToggle.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if NETFRAMEWORK
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Datadog.Trace.AppSec;
+using Datadog.Trace.AppSec.RcmModels.Asm;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.TestHelpers;
+using Datadog.Trace.Vendors.Newtonsoft.Json.Linq;
+using Xunit;
+using Xunit.Abstractions;
+
+#pragma warning disable SA1402 // File may only contain a single class
+#pragma warning disable SA1649 // File name must match first type name
+
+namespace Datadog.Trace.Security.IntegrationTests.Rcm;
+
+[Collection("IisTests")]
+public class AspNetMvc5AsmRulesToggleIntegratedWithSecurity : AspNetMvc5AsmRulesToggle
```

Comment:
I know nothing about global rules toggling so I honestly don't know, but isn't the asp version we run in orthogonal to this? Should we add that many testS?

Reply:
The toggling is about about turning blocking on / off. The way blocking works on different platforms is quite different, so I think it's good to have these tests. (Though blocking on the different platforms is tested in other places, but I think it's worth testing here again)

That said most RC features are fairly platform independent, @anna-git we should think carefully about whether Framework + Core tests are needed for each new RC feature.